presto-hive/src/main/java/com/facebook/presto/hive/LocationHandle.java

```diff
@@ -80,6 +85,11 @@ boolean isExistingTable()
         return isExistingTable;
     }
 
+    boolean isTemporaryTable()
```

Comment:
It is used in our internal LocationService implementation

Reply:
I will take a look at that PR as well. Ideally we don't want this since this is also quite confused (you already try to avoid to have both `existingTable` and `temporaryTable` : https://github.com/prestodb/presto/pull/12464/commits/8aa35995057a6d1e402e3739dbe82217c538cdac#r268791481 )

Reply:
I replaced the `isExistingTable` and the `isTemporaryTable` flags with the `LocationHandle#TableType{NEW, EXISTING, TEMPORARY}` enum.